json_feature.hpp

```diff
@@ -18,16 +18,18 @@ class JSONFeature {
     rapidjson::StringBuffer m_stream;
     writer_type m_writer;
     osmium::geom::RapidGeoJSONFactory<writer_type> m_factory;
+    std::string m_attr_prefix;
 
 public:
 
-    JSONFeature() :
+    JSONFeature(const std::string& attr_prefix) :
         m_stream(),
         m_writer(m_stream),
         m_factory(m_writer) {
         m_writer.StartObject();
         m_writer.String("type");
         m_writer.String("Feature");
+        m_attr_prefix = attr_prefix;
```

Comment:
Simple initializations of member variables of a C++ class should be done in the section above (like m_factory). Otherwise it will be initialized to the empty value first and then later overwritten.